packages/mdc-slider/test/component.test.ts

```diff
@@ -490,4 +490,29 @@ describe('MDCSlider', () => {
     document.body.removeChild(wrapper);
   });
 
+  it('correct value on pin value marker when value is 0', () => {
+    const root = getFixture();
+    Object.assign(root.style, {
```

Comment:
Can you change this to directly set properties instead of using Object.assign? That will fix IE11 issues and [match what we're doing elsewhere](https://github.com/material-components/material-components-web/blob/56a8bfef1b118817060891efd4d050e8d4b5a383/packages/mdc-slider/test/component.test.ts#L108-L111).

```ts
root.style.position = 'absolute';
root.style.left = '0';
root.style.top = '0';
root.style.width = '400px';
```

Reply:
Done.